src/systems/filecoin_mining/storage_mining/storage_miner_actor.go

```diff
@@ -597,16 +604,14 @@ func (st *StorageMinerActorState_I) _isSealVerificationCorrect(rt Runtime, onCha
 
 		// TODO: Make SealCfg sector.SealCfg from miner configuration (where is that?)
 		SealCfg_: &sector.SealCfg_I{
-			SectorSize_:     st.Info().SectorSize(),
+			SectorSize_:     sectorSize,
 			SubsectorCount_: st.Info().SubsectorCount(),
 			Partitions_:     st.Info().Partitions(),
 		},
 
-		// TODO: get Randomness sector.SealRandomness using onChainInfo.Epoch
-		//Randomness_:
-		// TODO: get InteractiveRandomness sector.SealRandomness using onChainInfo.InteractiveEpoch
-		//InteractiveRandomness_:
-		PieceInfos_: pieceInfos,
+		Randomness_:            sector.SealRandomness(rt.Randomness(onChainInfo.SealEpoch(), 0)),
+		InteractiveRandomness_: sector.InteractiveSealRandomness(rt.Randomness(onChainInfo.InteractiveEpoch(), 0)),
```

Comment:
It might be problematic to hard-code `0` as the offset here, in case other code accidentally uses `0` also. This likely merits further discussion, but for the moment it seems safest to me to treat these offsets in a similar fashion to MethodNums in the VM (in the sense that we should take special care to enforce their uniqueness).

Reply:
I think I must not understand the purpose/meaning of the offset here. Can you clarify?

Reply:
- the offset is used to derive randomness. think of `rt.Randomness(onChainInfo.InteractiveEpoch(), 0)` like `H(epoch || <randomness from vrf @ epoch> || 0)` <-- the 0 is an additional input that lets the caller derive different randomness values for the same epoch.